### PR TITLE
[Do not merge] Update and refresh orbit intro for licensing change

### PR DESCRIPTION
**Pull request by amarrazza**

### CODE REVIEW

**Review by TucksonDev — APPROVED** [review]
Awesome update. Thanks @amarrazza 🚀

**Comment by TucksonDev** [comment]
@cla-bot check

**Comment by TucksonDev** [comment]
@cla-bot check

**Review by symbolpunk — APPROVED** [review]
diagram updated!

File: arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md [hunk 1]
```diff
@@ -18,13 +18,12 @@ import PublicPreviewBannerPartial from './partials/_orbit-public-preview-banner-
...
-  - **Offer gas price reliability** to your end-users thanks to the dedicated throughput and traffic isolation that Orbit chains offer by default.
-  - **Permission access** to control who can read your chain's data and who can deploy smart contracts to your chain. Your chain can be completely permissionless like Ethereum and <a data-quicklook-from="arbitrum-one">Arbitrum One</a>, or you can implement your own permissions policies.
-  - **Collect fees using a token that you choose** to rapidly iterate on domain-specific mechanism designs and value capture opportunities.
+- Arbitrum Orbit is the **permissionless path** for launching customizable dedicated chains using Arbitrum technology.
+- Orbit Chains can be a <a data-quicklook-from='layer-2-l2'>Layer 2 (L2)</a> chain which settles directly to Ethereum, or a Layer 3 (L3) chain which can settle to one of Arbitrum's L2s, such as <a data-quicklook-from='arbitrum-one'>Arbitrum One</a>.
```
Comment (amarrazza) [inline arbitrum-docs/launch-orbit-chain/orbit-gentle-introduction.md:22]:
```suggestion
- Orbit Chains can be a <a data-quicklook-from='layer-2-l2'>Layer 2 (L2)</a> chain which settles directly to Ethereum, or a Layer 3 (L3) chain which can settle to any Ethereum L2, such as <a data-quicklook-from='arbitrum-one'>Arbitrum One</a>.
```

**Pull request merged**